README.md

```diff
@@ -13,35 +13,35 @@ Copyright (C) 2020 National Center for Atmospheric Research
 The fastest way to get started with MusicBox is with Docker. You will need to have [Docker Desktop](https://www.docker.com/get-started) and [git](https://git-scm.com) installed. Then, from a terminal window run:
 
 ```
-git clone https://github.com/NCAR/MusicBox2
-cd MusicBox2
+git clone --recurse-submodules https://github.com/NCAR/music-box
+cd music-box
 ```
 
-To set up the box model with mechanism 272 (Chapman chemistry) from the [Chemistry Cafe](https://chemistrycafe-devel.acom.ucar.edu/index.html#/Tags), run:
+To set up the box model with the Chapman chemistry mechanism from the [Public Chemistry Cafe Mechanism Store](https://www.acom.ucar.edu/cafe), run:
 
 ```
-docker build -t music-box-test . --build-arg TAG_ID=272
+docker build -t music-box-test . --build-arg TAG_ID=chapman
 docker run -it music-box-test bash
-cd build
+cd music-box/build
```

Comment:
the `build/` folder is in the root directory, so I think the original is correct here:
```
cd build
```

Reply:
You are correct.  The next pull request fixes this.